core/asset/asset.go

```diff
@@ -17,6 +17,11 @@ const (
 	Token    AssetType = 0x11
 )
 
+const (
+	MAX_PRECISION = 8
+	MIN_PRECISION = 0
```

Comment:
thanks, i have changed to CamelCase naming rule. could you helps to check it agian thanks.